aad/spring-cloud-azure-starter-active-directory/aad-resource-server-obo/pom.xml

```diff
@@ -36,6 +35,10 @@
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-web</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-webflux</artifactId>
+    </dependency>
```

Comment:
why do we need this?

Reply:
Shouldn't we stick to only of them here, spring-boot-starter-web or spring-boot-starter-webflux? Do we have a hard requirement on the WebClient here?

Reply:
Because this project need WebClient, I think.

Reply:
Yes. need WebClient. It can be replaced with RestTemplete, just a little more complicated.